### DISCUSSION TITLE
感谢大佬的模板，非常顺利地毕业了😁

### DISCUSSION THREAD

**Discussion by hqyyqh** [mengchaoheng/SCUT_thesis#7] [General]

**Comment by mengchaoheng** [comment]
哈哈哈，恭喜恭喜！可以愉快地玩耍啦！


| |
蒙超恒
|
|
***@***.***
|

签名由 网易邮箱大师 定制

在2021年06月08日 17:22，hqyyqh 写道：

**Comment by hqyyqh** [comment]
最近需要提交论文，论文末尾其实还应该加上 `\includepdf{答辩委员会对论文的评语.pdf}` 一项。
不过总体来说非常非常省心了。到最后还突然发现已经有【黑白，提交版】的超链接配置😁，贴心得不得了哈哈哈。

**Comment by zhuohoudeputao** [comment]
今年用这个模板也过审了🙋

**Reply by mengchaoheng** [reply]
哈哈其实送审回来之后根据博士们的结果有更新一点点模版。不过大体上无关紧要。更新包括：新增模版第四章，还有公式段前短后间距调整、表格与插图目录内容间距修改、定理环境斜体改为正体。

硕士们暂时没有反馈到模版问题。部分博士没用上诉环境的也没用反馈，甚至还得到好评。我也感谢大家共同的努力啦！毕业快乐！
